packages/datadog_tracking_http_client/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 
 
+## 1.5.0
```

Comment:
No need to add the version here. Our release scripts add the version when releasing.